ui/components/ui/logo/logo-moonpay.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const COLOR_MOONPAY_CIRCLES = '#7D00FF';
+
+const LogoMoonPay = ({
+  className,
+  size,
+  color = 'var(--color-text-default)',
+}) => {
+  return (
+    <svg
+      fill={color}
```

Comment:
Oh this is my bad sorry! I think `fill={color}` just needs to go in the path that the color concerns

```suggestion
```

Reply:
Removed on my local still looks good.

<img width="372" alt="Screen Shot 2022-03-16 at 7 38 58 AM" src="https://user-images.githubusercontent.com/8112138/158615427-04f5dc37-f83b-43a5-9214-f9f5bf9ff4d8.png">